### ISSUE TITLE
Error in logging

### ISSUE THREAD

**Issue by duncanokeyo** [tonyofrancis/Fetch#226]
fetch is reporting error for a different namespace. from the log below, you can see fetch reporting error for namespace batchdownloads yet the error is for singledownlaods. NOTE, i created two fetch instances and i enabled debugging

`E/batchDownloads: FileDownloader download:DownloadInfo(id=1352038749, namespace='singleDownloads', url='https://www.google.com/url?sa=t&source=web&rct=j&url=https://m.youtube.com/watch%3Fv%3DVbWfKVBpvZY&ved=2ahUKEwi0wtuJ-f3cAhWGx4UKHWzuATAQ3ywwAXoECAkQEA&usg=AOvVaw3fpnkGrFsqdv7_gpkraMdc', file='/storage/emulated/0/Android/data/com.dans.apps`

```
final FetchConfiguration fetchConfiguration = new FetchConfiguration.Builder(this)
                .enableRetryOnNetworkGain(true)
                .enableAutoStart(false)
                .enableLogging(BuildConfig.DEBUG)
                .enableMd5Check(false)
                .setNamespace("singleDownloads")
                .setDownloadConcurrentLimit(2).setProgressReportingInterval(500)
                .setHttpDownloader(new DownloadHelper(Downloader.FileDownloaderType.PARALLEL, false))
                .build();
        final FetchConfiguration fetchGroupConfiguration = new FetchConfiguration.Builder(this)
                .enableRetryOnNetworkGain(true)
                .setDownloadConcurrentLimit(4)
                .setProgressReportingInterval(500)
                .setNamespace("batchDownloads")
                .enableAutoStart(false)
                .enableLogging(BuildConfig.DEBUG)
                .enableMd5Check(false)
                .setHttpDownloader(new DownloadHelper(Downloader.FileDownloaderType.SEQUENTIAL, true))
                .build();
fetch = Fetch.Impl.getInstance(fetchConfiguration);
        fetchGroup = Fetch.Impl.getInstance(fetchGroupConfiguration);

```
`

**Comment by tonyofrancis** [comment]
@duncanokeyo I see the issue. Fetch has a default logger unless one is provided. The last fetch instance created namespace will return the existing. That's a bug. Will fix in next release. The workaround is provide the FetchConfiguration its logger.

**Comment by tonyofrancis** [comment]
fixed in 2.2.0-RC9
